### PR TITLE
1118 return 404 on not found

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#1118

### Dependencies

none

### Description

Return 404 on device not found.

Also update the dashboard's source w/ an additional widget to list API responses.

### Release Plan

- nothing special

### CODE REVIEW

File: packages/infra/dashboards/cw-dashboard-prod.json [hunk 1]
```diff
@@ -389,8 +389,8 @@
                 "annotations": {
                     "horizontal": [
                         {
-                            "label": "VolumeWriteIOPs >= 100_000 for 1 datapoints within 5 minutes",
-                            "value": 100000,
+                            "label": "VolumeWriteIOPs >= 5000 for 1 datapoints within 5 minutes",
```
Comment (RamilGaripov) [inline packages/infra/dashboards/cw-dashboard-prod.json:392]: Why are we changing this back to 5000? I thought we updated the threshold to 100k

**Review by RamilGaripov — APPROVED** [review]
Posted a question out of curiosity. Code lgtm

File: packages/infra/dashboards/cw-dashboard-prod.json (hunk 1 above)
Comment (leite08) [reply packages/infra/dashboards/cw-dashboard-prod.json:392]: I didn't update this section of the dashboard, was it updated to 100K when we updated CDK/infra?

Comment (leite08) [reply packages/infra/dashboards/cw-dashboard-prod.json:392]: oh, we likely updated the source of the dashboard but didn't update the dashboard on AWS... I'll try to fix this

**Pull request merged**